README.md

```diff
@@ -124,3 +128,71 @@ interpreting them with HIBA.
 libhiba provides convenience functions to parse and extract HIBA extensions
 directly from a certificate, but it does not verify the certificate. It assumes
 the certificate was validated by a prior stage.
+
+### Example CA
```

Comment:
It might actually make more sense to create a separate file here, and reference it in the README (sorry for coming back on my first idea).

The rationale:
- the example usage is longer than the rest of the file itself
- hiba-ca.sh is just one way of dealing with HIBA, there can be other CA integrations.